Firmware/LowLevel/README-Sound, DFPIS5V.md

```diff
@@ -0,0 +1,77 @@
+# Sound via "DFPlayer Mini"
+
+> **Warning**
+> Consider about switching your DFPlayer's VCC from 3.3V to 5V
+> (via solder jumper JP1 on your OpenMower MainBoard)
+
+**Explanation:**<br>
+I'm with sound since June 2023.<br>
+In the first month, I killed 2 Picos because his tiny Buck-Boost Converter "RT6150" cracked.<br>
+After a short correspondence with Clemens, he pointed me to the DFPlayer
+as a possible reason.<br>
+Because: By OM default design, the DFPlayer's VCC is 3.3V (via JP1),
+which is provided by the small Buck-Boost Converter on the Pico.<br>
+After I switched my DFPlayer's VCC to 5V (via JP1), I didn't lost any Pico anymore.<br>
+It's not confirmed yet if that really was the reason for my killed Pico's,
+and you're invited to validate the assumption by leaving your DFPlayer on 3.3V. **But be warned**, even if the Pico is cheap, it's awful to replace it! 
+
+### Update 10/13/2023
+
+As I was still in doubt if it's really necessary to switch DFPlayer's VCC to 5V,
+I did some measuring today:
+
+- Placed an 0.2Ω resistor (4W) within Pico's 3V3 output line
+- Measured with an oscilloscope the occuring voltage over the resistor:<br>
+  0.028 V<sub>AVG</sub> => divided by 0.2Ω = 140mA => looks fine<br>
+  0.195 V<sub>PP</sub> => divided by 0.2Ω = 975mA => hugh :-/ but this is V<sub>PP</sub>!
+- Within the Pico Datasheet it's written (somewhere), thats allowed to draw up to 300mA
+- The specs of the Buck-Boost Converter "RT6150" (used on the Pico) say:<br>
+  "*Up to 800mA Continuous Output Current*", as well as<br>
+  "*... current limit.*"
+
+End of October 2023, I discussed my doubts with Clemens and he answered:
+> The short peaks are also what worries me. The overcurrent protection will probably only take effect in the event of a longer overload. That's how you run it above the spec (even if for a short time) and that potentially breaks it at some point
+
+> **Important**
+> You may run `..._DFPIS5V` (to get full sound support), even if **not** switched to 5V, but we worry that you might kill your Pico's VREG on long time
+
+## Sound Buttons
+
+| Button<br>C500(B) | Button<br>RM-ECOW-V1.0.0 | Function |
+| ------ | -------- | ---
+| <kbd>Mon</kbd> | <kbd>4H</kbd> | Volume up |
+| <kbd>Tue</kbd> | <kbd>6H</kbd> | Volume down
+| <kbd>Wed</kbd> | <kbd>8H</kbd> | ~~Language switch (English, German)~~
```

Comment:
this doesn't work anymore

Reply:
Fixed/noticed in my readme.
Will check about the official docs (fully outdated regarding the official docs because I still use the first ones).
We probably also need to notice somewhere, that "save to flash" happen only once a minute?

Reply:
Took a while to get it running with Hugo. After I realized that docsys template is to old for recent Hugo versions, and downgraded to Hugo v0.110.0, all looks as expected (for me).

Integrated now all sound relevant infos to the official docs https://github.com/ClemensElflein/openmower.de/pull/33.
Or do you notice anything wrong or missing?